docs/cluster_management.md

```diff
@@ -2,45 +2,152 @@
 
 This doc is in progress.
 
+## Overview
+
+Kubernetes is an orchestration system for managing container clusters. This document provides basic instructions for setting up and managing the cluster and assumes familiarity with Kubernetes architecture and terminology.
```

Comment:
Done. Added a link to docs/admin (which links to this document) to main Kubernetes Readme.md